### ISSUE TITLE
Question regarding KL calculation

### ISSUE THREAD

**Issue by backpropper** [ericjang/gumbel-softmax#7]
In the `gumbel_softmax_vae_v2` notebook, for calculating the KL in case of Relaxed Prior, shouldn't the KL be `y * (log q_y - log p_y)`. So the product with `y` is missing?

**Comment by backpropper** [comment]
I see that its already a part the log_prob function.

**Comment by diegoantognini** [comment]
@backpropper  could you please explain what we don't have to multiply do ```p_y.log_prob(y).exp() * (q_y.log_prob(y) - p_y.log_prob(y))``` ?
